src/reference/asciidoc/gateway.adoc

```diff
@@ -254,17 +254,17 @@ public interface MyGateway {
 <1> Note that, in this example, the SpEL variable, `#this`, refers to the argument -- in this case, the value of `s`.
 
 The XML equivalent looks a little different, since there is no `#this` context for the method argument.
-However, expressions can refer to method arguments by using the `#args` variable, as the following example shows:
+However, expressions can refer to method arguments by using the `args` property for the `MethodArgsHolder` root object, as the following example shows:
```

Comment:
Suggest adding an anchor to `===== Expressions and "Global" Headers` and adding a link here to fully describe the holder.

See <<...>> for more information about this root object.